### DISCUSSION TITLE
GA4 OSS connector auth

### DISCUSSION THREAD

**Discussion by jwright12** [airbytehq/airbyte#40636] [Platform General Questions]
https://docs.airbyte.com/integrations/sources/google-analytics-data-api

Why is the service account auth approach specifically recommended for AirByte OSS GA4 connectors? Does OAuth not work for GA4 OSS?

**Comment by nataliekwong** [comment]
Hey there, the service account is the simplest way to authenticate in OSS. I would recommend using Cloud if you require OSS and minimal setup.

You can see our [API docs](https://reference.airbyte.com/reference/creating-oauth-sources) on how to set up your own OAuth in OSS.

Hope this answers your question!
